hotham/src/contexts/render_context.rs

```diff
@@ -71,35 +74,7 @@ pub struct RenderContext {
     pub primitive_map: HashMap<u32, InstancedPrimitive>,
 }
 
-pub struct Shaders {
-    pub vertex_shader: Vec<u32>,
-    pub fragment_shader: Vec<u32>,
-    pub compute_shader: Vec<u32>,
-}
-
-impl Default for Shaders {
```

Comment:
This default was here for a reason. 😛 `RenderContext` is not designed to be a generic renderer - it has very specific knowledge about the shaders and pipeline it's using to render them, so it makes sense to have strong defaults.